### PR TITLE
[AIP4] Update Simple Map To Save Gas

### PR DESCRIPTION
**Pull request by areshand**

### CODE REVIEW

File: aips/aip-4.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+aip: 4
+title: Update Simple Map To Save Gas
+author: areshand
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/15
+Status: Review
```
Comment (sherry-x) [inline aips/aip-4.md:6]: let's put this as draft stage and we will update to review once we get enough maintainer support

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (sherry-x) [inline .gitignore:1]: do you intend to commit this?

Comment (areshand) [reply .gitignore:1]: yes. otherwise, it is easy to commit this folder unintentionally.

File: aips/aip-4.md (hunk 1 above)
Comment (areshand) [reply aips/aip-4.md]: updated

**Review by sherry-x — APPROVED** [review]

**Comment by sherry-x** [comment]
Looks like maintainer has discussed this internally thus approving this to be a draft AIP to receive future feedback and moving forward.

**Pull request merged**